src/Console/Commands/FindRiskyDatabaseColumns.php

```diff
@@ -218,4 +210,15 @@ private function formatBytes(int $size, int $precision): string
         $suffix = $suffixes[$index];
         return round(1024 ** ($base - floor($base)), $precision).$suffix;
     }
+    protected function getTableSize($connection, string $table)
```

Comment:
I agree. It would make more sense to change the method to private. I'll also add the return and parameter types as you suggested. Thanks for the catch!